src/App/routes.tsx

```diff
@@ -1101,6 +1101,36 @@ const fieldReportFormNew = customWrapRoute({
     },
 });
 
+const fieldReportsFormDetails = customWrapRoute({
+    path: 'field-reports/:fieldReportId',
+    component: {
+        render: () => import('#views/FieldReportDetails'),
+        props: {},
+    },
+    parent: root,
+    wrapperComponent: Auth,
+    context: {
+        title: 'All Field Reports',
+        visibility: 'anything',
```

Comment:
Maybe we need to name the title suitably. Since it shows a specific field report.